app.js

```diff
@@ -272,12 +272,18 @@ models.sequelize.sync().then(function () {
   } else {
     throw new Error('server still not ready after db synced')
   }
+}).catch(err => {
+  logger.error('Can\'t sync database');
```

Comment:
```suggestion
  logger.error('Can\'t sync database')
```

Travis CI reported here has a extra semicolon.
https://travis-ci.com/github/hackmdio/codimd/jobs/318483790

Reply:
fixed